src/src/cpp/environments/classic_control/MountainCar.h

```diff
@@ -12,83 +12,94 @@
 #include <GL/glut.h>
 #endif
 
-#define STATE_SIZE 4
-
-using namespace std;
+// Number of state variables in the mountain car environment
+constexpr int kMountainCarStateSize = 4;
 
 class MountainCar : public ClassicControlEnv {
    protected:
-    const double min_position = -1.2;
-    const double max_position = 0.6;
-    const double max_speed = 0.07;
-    const double goal_position = 0.5;
-    const double goal_velocity = 0.0;
-    const double force = 0.001;
-    const double gravity = 0.0025;
-
-    // state array indexing
-    const int _position = 0;
-    const int _velocity = 1;
+    // Environment constants
+    static constexpr double kMinPosition = -1.2;
+    static constexpr double kMaxPosition = 0.6;
+    static constexpr double kMaxSpeed = 0.07;
+    static constexpr double kGoalPosition = 0.5;
+    static constexpr double kGoalVelocity = 0.0;
+    static constexpr double kForce = 0.001;
+    static constexpr double kGravity = 0.0025;
+
+    // State array indices
+    static constexpr int kPositionIndex = 0;
```

Comment:
for state arrays, can you make them enums to make em cleaner, like what I have [here](https://github.com/TPGEngine/tpg/pull/126/files#diff-10fc356e85afaa26cc0b3c6bf9d79e6f77a2cf3eb550454148c23e6971ec5fabR36-R42)